Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
+
+RUN pip install pipenv
+WORKDIR /usr/src/app
+COPY Pipfile .
+COPY Pipfile.lock .
+RUN pipenv install --system  --ignore-pipfile
```

Comment:
The `--ignore-pipfile` means it's installing the dependencies from the `Pipfile.lock` file, so you don't need the `Pipfile` in the docker.

Reply:
If I copy only the Pipfile.lock without the Pipfile, the "pipenv install --system  --ignore-pipfile" does not work.
anyway I copied all project directory to the docker image, because I still need to figure out how to copy a specific directors and keep the structure formation.

Reply:
Actually do that, copy it all, but use the `.dockerignore` file (Google it..)

Reply:
OK WOW! this is cool feature `.dockerignore` did it..
can I marge this branch to the master and move to the next issue?
on the next issue I need to open a new branch??

Reply:
If you think the PR is ready to be reviewed write me a comment that says it's ready..
I'll review it now..